src/keystore/providers/helpers.ts

```diff
@@ -11,6 +11,9 @@ export const buildPersistenceFromOptions = async (
   opts: KeystoreProviderOptions,
   keys: PrivateKeyBundleV1 | PrivateKeyBundleV2
 ) => {
+  if (!opts.persistConversations) {
```

Comment:
This is a helper that all existing JS keystore implementations consume, but other keystore impls have their own responsibility to process this flag correctly?

Reply:
That's right, but it's actually the `KeystoreProvider` implementations that have to respect this flag. The Keystore takes whatever persistence is passed in to it from the provider.

Reply:
Ah gotcha